src/main/scala/dpla/ingestion3/mappers/providers/SiMapping.scala

```diff
@@ -42,6 +42,46 @@ class SiMapping extends XmlMapping with XmlExtractor {
 
   override def dplaUri(data: Document[NodeSeq]): ZeroToOne[URI] = mintDplaItemUri(data)
 
+  override def edmRights(data: Document[NodeSeq]): ZeroToMany[URI] = {
+    val edmRightsReadable = extractStrings( data \\ "objectRights")
+
+    edmRightsReadable.length match {
+      case 0 => rightsFromMedia(extractStrings(data \\ "media" \ "usage" \ "access"))
+      case 1 => edmRightsReadable.map(lookupRightsText).map(URI)
+      case _ => Seq()
+    }
+  }
+  private def lookupRightsText(str: String): String =
+    str.toLowerCase.trim match {
+      case "cc0" => "http://rightsstatements.org/vocab/NoC-US/1.0/" // TODO confirm with Dominic
+      case "in copyright" => "http://rightsstatements.org/vocab/InC/1.0/"
+      case "in copyright - eu orphan work" => "http://rightsstatements.org/vocab/InC-EDU/1.0/"
+      case "in copyright -non - commercial use permitted" => "http://rightsstatements.org/vocab/InC-EDU/1.0/"
+      case "in copyright -educational use permitted" => "http://rightsstatements.org/vocab/InC-NC/1.0/"
+      case "in copyright - rights- holder(s) unlocatable or unidentifiable" => "http://rightsstatements.org/vocab/InC-RUU/1.0/"
+      case "no copyright - contractual restrictions" => "http://rightsstatements.org/vocab/NoC-CR/1.0/"
+      case "no copyright - non-commercial use only" => "http://rightsstatements.org/vocab/NoC-NC/1.0/"
+      case "no copyright - other known legal restrictions" => "http://rightsstatements.org/vocab/NoC-OKLR/1.0/"
+      case "no copyright - united states" => "http://rightsstatements.org/vocab/NoC-US/1.0/"
+      case "copyright not evaluated" => "http://rightsstatements.org/vocab/CNE/1.0/"
+      case "copyright undetermined" => "http://rightsstatements.org/vocab/UND/1.0/"
+      case "no known copyright" => "http://rightsstatements.org/vocab/NKC/1.0/"
+      case _ => ""
+    }
```

Comment:
Just sanity checking that the whitespace around dashes is correct in lines 59-61 b/c it looks a bit wonky, e.g. `in copyright -non - commercial`

Reply:
Thanks!